package.json

```diff
@@ -30,52 +30,51 @@
     "e2e": "cross-env TS_NODE_PROJECT=tsconfig.json xvfb-maybe mocha --require ts-node/register/transpile-only --timeout 30000 ./test/e2e/**/*.ts"
   },
   "dependencies": {
-    "@sentry/browser": "4.0.0-rc.1",
-    "@sentry/core": "4.0.0-rc.1",
-    "@sentry/hub": "4.0.0-rc.1",
-    "@sentry/node": "4.0.0-rc.1",
-    "@sentry/minimal": "4.0.0-rc.1",
-    "@sentry/utils": "4.0.0-rc.1",
-    "@sentry/types": "4.0.0-rc.1",
+    "@sentry/browser": "4.0.0",
```

Comment:
We could make these `^4.0.0` so that bug fixes in all the core stuff doesn't require bumping dependencies so often?

Reply:
Good point!